client/state/lib/middleware.js

```diff
@@ -227,6 +230,15 @@ const handler = ( dispatch, action, getState ) => {
 				if ( desktopEnabled ) {
 					updateSelectedSiteForDesktop( dispatch, action, getState );
 				}
+
+				const state = getState();
+				const siteId = getSelectedSiteId( state );
+
+				if ( ! fetchingATStatus && hasSitePendingAutomatedTransfer( state, siteId ) ) {
+					fetchingATStatus = true;
```

Comment:
`fetchingATStatus` should have distinct value for each `siteId`. It's a rare edge case, but if you create and transfer a second site without reloading Calypso, it will not work 🙂 

It would be great if this "already fetching" flag was somewhere inside `fetchAutomatedTransferStatus`. Then I could call it as many times as I want and it would mean "start the loop if it's not already running".

Reply:
Fixed in 2524807.